src/components/App.js

```diff
@@ -48,6 +49,7 @@ const App = (props) => {
 						<Route exact path={routes.PROFILE} component={ProfilePage} />
 						<Route exact path={routes.USER_PAGE} component={UserPage} />
 						<Route exact path={routes.ORGANISATION_PAGE} component={OrganisationPage} />
+            <Route exact path={routes.ABOUT_PAGE} component={AboutPage} />
```

Comment:
Minor issue with tabs v spaces I think, not a big problem as it's kind of inconsistent across the board right now, will look into adding an `.editorconfig` / ESLint or something

Reply:
Oh weird! That must have been my linter, which was also the source of the deleted whitespace. Will fix this.

Reply:
Well I just managed to make it worse. Is this going to break it, or shall I leave it for now